js/ui/grid_core/ui.grid_core.column_fixing.js

```diff
@@ -676,7 +688,7 @@ const RowsViewFixedColumnsExtender = extend({}, baseFixedColumns, {
                         scrollTop = scrollable.scrollTop();
                         scrollable.scrollTo({ y: scrollTop - e.delta });
 
-                        if(scrollable.scrollTop() > 0 && (scrollable.scrollTop() + scrollable.clientHeight()) < (scrollable.scrollHeight() + that.getScrollbarWidth())) {
+                        if(scrollable.scrollTop() > 0 && (scrollable.scrollTop() + scrollable.clientHeight()) < (scrollable.scrollHeight() + this.getScrollbarWidth())) {
                             return false;
                         }
```

Comment:
Let's move this condition to the separate variable:

```suggestion
                        const scrollableTop = scrollable.scrollTop() + scrollable.clientHeight();
                        const scrollableHeight = scrollable.scrollHeight() + this.getScrollbarWidth();
                        // I'm not sure about the name of this variable
                        const isPreventDefault = scrollable.scrollTop() > 0 && scrollableTop < scrollableHeight;
                        
                        if (isPreventDefault) {
                          return false;
                        }
```